lafourche/parser/osmparser.py

```diff
@@ -65,8 +66,8 @@ def get_node_registry(self, tree_root: ElementTree) -> {int: Node}:
                                   node.tag, node.attrib.get('id'), node.attrib.get('lat'), node.attrib.get('lon'))
                 continue
             else:
-                node_registry[node.attrib.get('id')] = Node(node.attrib.get('id'), node.attrib.get('lon'),
-                                                            node.attrib.get('lat'))
+                node_registry[node.attrib.get('id')] = Node(node.attrib.get('id'),
+                                                            Geopoint(node.attrib.get('lon'), node.attrib.get('lat')))
```

Comment:
this is a bug because Geopoint parameters should be float instead of str here

Reply:
Fixed in #13